source/draw/gpu/opengl/GraphicBufferYuv420Semiplanar.ooc

```diff
@@ -35,8 +35,7 @@ GraphicBufferYuv420Semiplanar: class extends RasterYuv420Semiplanar {
 		ptr := _buffer lock()
 		_buffer unlock()
 		length := 3 * this _stride * size height / 2
-		byteBuffer := ByteBuffer new(ptr, length, func (buffer: ByteBuffer))
-		super(byteBuffer, size, _stride, _uvOffset)
+		super(ByteBuffer new(ptr, length), size, _stride, _uvOffset)
```

Comment:
Nice. This change alone can be the reason why it should be in the master branch.